src/tag.rs

```diff
@@ -298,6 +303,20 @@ impl<'a> Tag {
         });
     }
 
+    /// Returns STREAMINFO block if exists
+    pub fn get_streaminfo(&self) -> Option<&StreamInfo> {
+        match self.blocks.get(0) {
```

Comment:
I would rather have this iterate over the blocks and return the first StreamInfo block. In the event it is the first block, it should have the same time complexity as this, while also allowing for cases where it might not be the first block.

Reply:
Done